Formula/r/rails-new.rb

```diff
@@ -0,0 +1,25 @@
+# typed: false
+
+class RailsNew < Formula
+  desc "Create Rails projects with Ruby installed"
+  homepage "https://github.com/rails/rails-new"
+  url "https://github.com/rails/rails-new/archive/refs/tags/v0.5.0.tar.gz"
+  sha256 "9a309ea0d3f7b7c10327aba30e919bab30efc3bdffc2fcedaa54ce23d9e4ae42"
+  license "MIT"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  def caveats
+    <<~TEXT
+      You need to install 'docker' in your environment first.
+    TEXT
+  end
+
+  test do
+    assert_equal "rails-new 0.5.0\n", shell_output("#{bin}/rails-new --version")
```

Comment:
also need a more functional test

```suggestion
    assert_match version.to_s, shell_output("#{bin}/rails-new --version")
```